lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -74,12 +74,21 @@ describe('asset-saver helper', () => {
       fs.unlinkSync(filename);
     });
 
-    it('screenshots file saved to disk with data', () => {
-      const ssFilename = 'the_file-0.screenshots.html';
-      const ssFileContents = fs.readFileSync(ssFilename, 'utf8');
+    it('screenshots html file saved to disk with data', () => {
+      const ssHTMLFilename = 'the_file-0.screenshots.html';
+      const ssFileContents = fs.readFileSync(ssHTMLFilename, 'utf8');
       assert.ok(/<!doctype/gim.test(ssFileContents));
-      assert.ok(ssFileContents.includes('{"timestamp":674089419.919'));
-      fs.unlinkSync(ssFilename);
+      const expectedScreenshotContent = '{"timestamp":674089419.919';
+      assert.ok(ssFileContents.includes(expectedScreenshotContent), , 'unexpected screenshot html');
+      fs.unlinkSync(ssHTMLFilename);
+    });
+
+    it('screenshots json file saved to disk with data', () => {
+      const ssJSONFilename = 'the_file-0.screenshots.json';
+      const ssFileContents = fs.readFileSync(ssJSONFilename, 'utf8');
+      const expectedScreenshotContent = '"timestamp": 674089419.919,';
+      assert.ok(ssFileContents.includes(expectedScreenshotContent), 'unexpected screenshot json');
```

Comment:
maybe we can be a bit stronger with our assertions here, maybe that it parses and it has the timestamp property?

Reply:
benim sitem çok yavaş : (( bana yardım edermisiniz http://ankaraotokurtarici.com